app/frontend/src/cljs/teet/ui/common.cljs

```diff
@@ -592,3 +593,20 @@
         [Paper
          (mapc (r/partial context-menu-item toggle!)
                (remove nil? items))]]]]]))
+
+(defn date-label-component
+  "Shows :label with formatted date from :value with an info icon and :tooltip"
+  [{value :value label :label tooltip :tooltip}]
+  [Grid
```

Comment:
Why is this a grid? If you only have 1 grid item the whole grid is unnecessary.

Reply:
Grid is used to apply alignment in the current structure of elements. Changing to :div causes errors applying flex alignment.